.github/workflows/helm.yaml

```diff
@@ -0,0 +1,62 @@
+name: Validate Chart
+
+on:
+  push:
+    paths:
+      - "chart/**"
+  pull_request:
+    paths:
+      - "chart/**"
+
+jobs:
+  lint-chart:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        chartName:
+          - openfaas
+          - cron-connector
+          - kafka-connector
+          - mqtt-connector
+          - nats-connector
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: azure/setup-helm@v1
+      - name: Helm Lint
+        working-directory: chart
+        run: helm lint ${{ matrix.chartName }}
+
+  kubeval-chart:
+    runs-on: ubuntu-latest
+    needs:
+      - lint-chart
+    strategy:
+      matrix:
+        k8s:
+          - v1.16.8
+          - v1.17.4
+          - v1.17.17
+          - v1.18.6
+          - v1.18.9
+          - v1.20.4
+          - v1.21.1
```

Comment:
@alexellis  we just need to pick a list of k8s versions we want to support/test  I just grabbed these from the workflow we use at work, which is basically just a lit of versions we currently deal with, i don't think there is a special meaning to this specific list